src/annotator/test/integration/hypothesis-injector-test.js

```diff
@@ -9,13 +8,35 @@ describe('HypothesisInjector integration test', () => {
 
   const sandbox = sinon.createSandbox();
   const config = {
-    clientUrl: 'data:,', // empty data uri
+    clientUrl: 'data:,Hypothesis', // empty data uri
   };
 
-  const waitForFrameObserver = cb => {
-    const wait = DEBOUNCE_WAIT + 10;
-    return setTimeout(cb, wait);
-  };
+  function waitForFrameObserver() {
+    return new Promise(resolve => setTimeout(resolve, DEBOUNCE_WAIT + 10));
+  }
+
+  function getHypothesisScript(iframe) {
+    return iframe.contentDocument.querySelector(
+      'script[src="data:,Hypothesis"]'
+    );
+  }
+
+  function createHypothesisInjector() {
+    const hypothesisInjector = new HypothesisInjector(
```

Comment:
```suggestion
    const injector = new HypothesisInjector(
```

Since this is just a local variable my preference would be to use a slightly more succinct name.